pkg/apis/batch/types.go

```diff
@@ -39,6 +39,15 @@ type Job struct {
 	Status JobStatus `json:"status,omitempty"`
 }
 
+func (j *Job) IsJobFinished() bool {
```

Comment:
We usually don't keep those in `types.go` files, can you create a `resource_helpers.go` in `pkg/apis/batch` and move it there. Similarly to how it's done in `pkg/api`?